timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -1202,23 +1202,30 @@ void declare_barostat(py::module &m) {
     std::string pyclass_name = std::string("MonteCarloBarostat");
     py::class_<Class, std::shared_ptr<Class>>(m, pyclass_name.c_str(), py::buffer_protocol(), py::dynamic_attr())
         .def(
-            py::init(
-                [](const int N,
-                   const double pressure,
-                   const double temperature,
-                   std::vector<std::vector<int>> group_idxs,
-                   const int frequency,
-                   std::vector<std::shared_ptr<timemachine::BoundPotential>> bps,
-                   const int seed) { return new Class(N, pressure, temperature, group_idxs, frequency, bps, seed); }),
+            py::init([](const int N,
+                        const double pressure,
+                        const double temperature,
+                        std::vector<std::vector<int>> group_idxs,
+                        const int frequency,
+                        std::vector<std::shared_ptr<timemachine::BoundPotential>> bps,
+                        const int seed,
+                        const bool adaptive) {
+                return new Class(N, pressure, temperature, group_idxs, frequency, bps, seed, adaptive);
+            }),
             py::arg("N"),
             py::arg("pressure"),
             py::arg("temperature"),
             py::arg("group_idxs"),
             py::arg("frequency"),
             py::arg("bps"),
-            py::arg("seed"))
+            py::arg("seed"),
+            py::arg("adaptive") = true)
         .def("set_interval", &Class::set_interval, py::arg("interval"))
         .def("get_interval", &Class::get_interval)
+        .def("set_volume_scaling", &Class::set_volume_scaling, py::arg("volume_scaling"))
```

Comment:
nit: consider `py::arg("volume_scale_factor")` if suggestion to change the getter and setter names is implemented

Reply:
Done in [5c39c88](https://github.com/proteneer/timemachine/pull/1152/commits/5c39c8893119ec43434328d333b41ae638a5cf05)